recipes/libgeoip/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "GeoIP" %}
+{% set version = "1.6.11" %}
+{% set sha256 = "b0e5a92200b5ab540d118983f7b7191caf4faf1ae879c44afa3ff2a2abcdb0f5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/maxmind/geoip-api-c/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - Makefile.vc.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - if not exist %LIBRARY_INC%\\{{ name }}.h exit 1     # [win]
+    - if not exist %LIBRARY_LIB%\\{{ name }}.lib exit 1          # [win]
+    - if not exist %LIBRARY_BIN%\\{{ name }}.dll exit 1          # [win]
```

Comment:
GeoIP.dll is created, but not installed. You'll have to manually copy it to `%LIBRARY_BIN%`